src/ext/java/org/opentripplanner/ext/transmodelapi/model/plan/ElevationProfileStepType.java

```diff
@@ -0,0 +1,71 @@
+package org.opentripplanner.ext.transmodelapi.model.plan;
+
+import graphql.Scalars;
+import graphql.schema.DataFetchingEnvironment;
+import graphql.schema.GraphQLFieldDefinition;
+import graphql.schema.GraphQLObjectType;
+import graphql.schema.GraphQLTypeReference;
+import java.util.List;
+import java.util.Objects;
+import org.opentripplanner.model.plan.ElevationProfile;
+
+public class ElevationProfileStepType {
+
+  private static final String NAME = "ElevationProfileStep";
+  public static final GraphQLTypeReference REF = new GraphQLTypeReference(NAME);
+
+  static String makeDescription(String name) {
+    return """
+       The %s's elevation profile.
+                 
+       **Notes**
+        - All elevation values, including the first one, are relative to the sea level.
+        - Negative values are possible for places below sea level.
+        - The data includes both the start end end coordinate of the %s.
+       """.formatted(
+        name,
+        name
+      )
```

Comment:
```suggestion
    return """
       The %s's elevation profile. All elevation values, including the first one, are in meters 
       above sea level. The elevation is negative for places below sea level. The profile 
       includes both the start and end coordinate.
       """.formatted(name)
```
This might need prettier formatting...